.github/workflows/bench.yml

```diff
@@ -51,8 +52,10 @@ jobs:
         fuseVersion: 2
         libunwind: true
         fio: true
-    - name: Set up stable Rust
-      uses: dtolnay/rust-toolchain@stable
+    - name: Set up Rust toolchain
+      uses: dtolnay/rust-toolchain@master
```

Comment:
My suggestion was to remove this step entirely, not to revert to `stable`

Reply:
Alternative: we could look at some other actions, e.g. https://github.com/marketplace/actions/rust-toolchain-toml-install

Reply:
Agh, I misunderstood. I think `dtolnay/rust-toolchain` installs `rustup`/`cargo`. We might look installing it in `.github/actions/install-dependencies` step, not sure if there are any downsides.

Reply:
I was wondering about jumping to a new action, I saw this one that looks well maintained: https://github.com/actions-rust-lang/setup-rust-toolchain

Reply:
Updated to use `actions-rust-lang/setup-rust-toolchain`. I had to override `rustflags` option though, it's `-D warnings` by default which turns all warnings into compile errors, and `vendor/fuser` has lots of warnings. It'd be nice to get rid of those warnings eventually, but not turning this on should be fine for now.

This new action also surfaces warnings on PRs, e.g., scroll to the bottom on https://github.com/awslabs/mountpoint-s3/pull/1073/files. We can turn this of it we find it noisy, but eventually I'd like to address these warnings as well.